manifests/plugin/foreman_wreckingball.pp

```diff
@@ -0,0 +1,5 @@
+# Installs foreman_wreckingball plugin
```

Comment:
Looks like the file should be named `wreckingball.pp` instead of `foreman_wreckingball.pp`